src/corner_kick/tslint.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": ["tslint:recommended", "tslint-react"],
+    "linterOptions": {
+        "exclude": ["config/**/*.js", "node_modules/**/*.ts", "coverage/lcov-report/*.js"]
+    },
+    "rules": {
+        "max-line-length": [true, 90],
+        "no-console": false,
+        "jsx-no-lambda": false,
+        "semicolon": [true, "always", "ignore-bound-class-methods"],
+        "object-literal-sort-keys": [true, "shorthand-first"],
+        "variable-name": false,
+        "match-default-export-name": false,
+        "import-name": false,
+        "quotemark": [true, "single", "jsx-double"]
+    }
```

Comment:
Is this file duplicating some work done in `.prettierc`? What's the difference between them?

Reply:
`.prettierc` and the Prettier library allows for the modification of the source code to match specific linting rules. It is somewhat limited in the types of modifications it can apply.
`tslint.json` and the TSLint library is an extensive linting library. It will not, however, modify the source code. Rather, it will fail the build if a rule is not respected.

TSLint is essential. Prettier is a nice to have to improve the development experience.